cross/libvpx/Makefile

```diff
@@ -46,6 +46,60 @@ ifeq ($(findstring $(ARCH),$(PPC_ARCHES)),$(ARCH))
 CONFIGURE_ARGS += --target=generic-gnu
 endif
 
+# Removal of GCC additional warning flags required at least for ppc853x-5.2 qoriq-5.2
```

Comment:
...or maybe you meant: if you split out the block of code this way, it's clearer wrt documenting? Could go either way, I guess ;)
Anyway, If I see a block of code doing arch-specific things, I would probably integrate new  code into the existing block rather than adding a new block.